.processes/bounty.md

```diff
@@ -0,0 +1,81 @@
+# Bounty Processes
+
+The HOPR Association _tech_ members, _community_ members, and [Project Owner](#legend) have agreed on the following bounty processes.
+The purpose of these processes is to follow a consistent way of creating, funding, and publishing bounties at HOPR.
+
+## The bounty
+
+A bounty usually serves multiple goals.
+
+1. Takes away workload from the _tech_ team as it allows external developers improve the HOPR protocol and the various applications/tools we maintain.
+2. Gives the _community_ team the opportunity to showcase HOPR's involvement in open source.
+3. Allows us to build a relationship between HOPR and the bounty hunter (tech community).
+
+The primary goal of a bounty creator is to take into account all 3 aspects, and depending on the bounty, adjust accordingly.
```

Comment:
Suggestions: "The bounty creator must consider all three factors and potentially adjust their bounty accordingly."

Possible addition: "Bounties do not have to address all three goals equally, and some bounties may address only two, but a bounty which only addresses one of these goals is likely a bad fit for this program and should be reconsidered."